praw/models/reddit/subreddit.py

```diff
@@ -2209,8 +2209,10 @@ def modmail_conversations(
 
         .. code-block:: python
 
-           for message in reddit.subreddit('all').mod.stream.modmail_conversations():
-               print("From: {}, To: {}".format(message.owner, message.participant))
+           subreddit = reddit.subreddit('all')
```

Comment:
Let's leave this as it was please, because the resulting example still adheres to the code style.